src/components/deals/loading/LoadingPreview.tsx

```diff
@@ -1,11 +1,65 @@
+import Section, { SECTION_STYLE, SECTION_WIDTH } from '@/components/Section'
 import LoadingDealImage from './LoadingDealImage'
-
+import LoadingTagsList from './LoadingTagsList'
+import LoadingText, {
+  LOADING_TEXT_STYLE,
+  LOADING_TEXT_WIDHTS,
+} from './LoadingText'
 export default function LoadingPreview() {
   return (
-    <div className="flex flex-col items-start gap-10 text-white xl:flex-row xl:items-center">
-      <div className="align-center relative aspect-video w-full max-w-[600px] self-center">
-        <LoadingDealImage />
-      </div>
+    <div>
+      <Section
+        width={SECTION_WIDTH.SM}
```

Comment:
If possible, can you explain why you're mixing TailwindCSS with props that take enums? I find the multiple imports and inconsistent pattern here a bit confusion.

Reply:
In my mind they serve different purposes. The intent is to have a consistent set of options for width for the developer to choose from instead of writing those tailwind classes directly. These obviously get translated to translated to Tailwind classes in the component itself. This way, if those every need to change, we can change them in one place. Passing the `className` prop allows the developer to do any further customization that's needed, although that shouldn't be very often (hopefully).